orchestration/cleanup_agents.sh

```
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Simple cleanup script for completed agent worktrees and old results
+
+set -e
+
+echo "🧹 Orchestration Agent Cleanup Script"
+echo "====================================="
+
+# Configuration
+RESULT_DIR="/tmp/orchestration_results"
+MAX_AGE_HOURS=24
```

Shellcheck (0.10.0)

[warning] MAX_AGE_HOURS appears unused. Verify use (or export if used externally).

(SC2034)